src/bridges/zora/ZoraBridge.sol

```diff
@@ -8,7 +8,7 @@ import {ErrorLib} from "../base/ErrorLib.sol";
 import {BridgeBase} from "../base/BridgeBase.sol";
 import {AddressRegistry} from "../registry/AddressRegistry.sol";
 
-// Get function definition from: https://github.com/ourzora/v3/blob/1d4c0c951ccd5c1d446283ce6fef3757ad97b804/contracts/modules/Asks/V1.1/AsksV1_1.sol.
+// https://docs.zora.co/docs/smart-contracts/modules/Asks/zora-v3-asks-v1.1
 contract ZoraAsk {
```

Comment:
you can specify these zora contracts as `interfaces` instead of `contracts` https://docs.soliditylang.org/en/v0.8.17/contracts.html#interfaces